image_client.py

```diff
@@ -150,24 +150,12 @@ def check_image_db_if_filename_imported(self, collection, filename, exact=False)
             'search_type': 'filename',
             'token': self.generate_token(filename)
         }
-        return self.decode_response(params)
 
-    # works for full filepath and original extension. "exact" does a sql "like" operation
```

Comment:
this function was an identical duplicate of an existing function in image_client